src/Mod/PartDesign/Gui/ViewProviderShapeBinder.cpp

```diff
@@ -259,7 +289,8 @@ void ViewProviderSubShapeBinder::setupContextMenu(QMenu* menu, QObject* receiver
     act = menu->addAction(QObject::tr("Synchronize"), receiver, member);
     act->setData(QVariant((int)0));
     act = menu->addAction(QObject::tr("Select bound object"), receiver, member);
-    act->setData(QVariant((int)1));
+    act->setData(QVariant((int)0x81));

```

Comment:
Is this just a random number?

Reply:
Yes. Just some big number to not conflict with those defined in `Gui::ViewProvider`.

Reply:
Thanks - I see now that it doesn't matter, all of the data elements have been replaced by enums in the master. Sorry for the noise...